tests/command.bats

```diff
@@ -28,6 +28,19 @@ export DOCKER_IMAGE=packer-ami-copy
   assert_failure
 }
 
+@test "executes prep_commands correctly" {
+  export BUILDKITE_PLUGIN_PACKER_AMI_COPY_PACKER_TEMPLATE=$PWD/tests/test-template.json
+  export BUILDKITE_PLUGIN_PACKER_AMI_COPY_PACKER_COMMANDS_0=validate
+  export BUILDKITE_PLUGIN_PACKER_AMI_COPY_PACKER_COMMANDS_1=build
+  export BUILDKITE_PLUGIN_PACKER_AMI_COPY_PREP_COMMANDS_0="echo test message 123"
+  export BUILDKITE_PLUGIN_PACKER_AMI_COPY_PREP_COMMANDS_1="exit 1"
+
+  run $PWD/hooks/command
+
+  assert_failure
+  assert_output --partial "test message 123"
+}
```

Comment:
It looks like there's no tests in this repo for the happy path. Can we stub out calls to `docker` and `packer` sufficiently to make that possible?

Reply:
I'm just reading the BATS docs now to figure out how stubbing works - what seemed obvious (using a function or an alias) doesn't appear to work

Reply:
Now have a test that expects the script to finish with success

Reply:
Awesome